src/BuiltInTools/dotnet-watch.slnf

```diff
@@ -1,12 +1,12 @@
 {
   "solution": {
-    "path": "..\\..\\..\\sdk.sln",
+    "path": "..\\..\\sdk.sln",
     "projects": [
       "src\\BuiltInTools\\BrowserRefresh\\Microsoft.AspNetCore.Watch.BrowserRefresh.csproj",
       "src\\BuiltInTools\\DotNetDeltaApplier\\Microsoft.Extensions.DotNetDeltaApplier.csproj",
       "src\\BuiltInTools\\DotNetWatchTasks\\DotNetWatchTasks.csproj",
+      "src\\BuiltInTools\\HotRestart\\Microsoft.Extensions.HotRestart.csproj",
```

Comment:
Microsoft.Extensions? What are the other sdk things called?

Reply:
I don't think there's a real theme since it ships a mix of random compiler, msbuild, nuget, and aspnetcore tools. Within dotnet-watch, the names are a bit of a hodgepodge:

* dotnet-watch.dll - the tool
* Microsoft.AspNetCore.Watch.BrowserRefresh.dll - the browser refresh middleware. VS references this assembly by name, so we're stuck with it.
* DotnetWatchTasks.dll - MSBuild tasks used by dotnet-watch
* Microsoft.Extensions.DotNetDeltaApplier - the startup hook supporting hot reload. VS currently references this assembly by name
* Microsoft.Extensions.HotRestart.dll - the new one